kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -432,7 +432,18 @@ void AddGeometricalUtilitiesToPython(pybind11::module &m)
         .def_static("CalculateShapeFunctions", [](const Matrix& rPoints, const array_1d<double,3>& rX, const double h, Vector& rN, DenseQRPointerType pDenseQR){
             return RBFShapeFunctionsUtility::CalculateShapeFunctions(rPoints, rX, h, rN, pDenseQR);})
         ;
+    
```

Comment:
See my comment about the Kratos naming (change `MPC` to `MasterSlaveConstraint`).